src/app/showcase/showcase.component.html

```diff
@@ -11,54 +11,67 @@ <h1>Showcase</h1>
     <aside fxFlex="0 1 20%"></aside>
 
     <div fxLayout="column" fxLayoutAlign="center" fxFlex="1 auto">
-
-      <!-- Picture showcase -->
-      <div class="section">
-        <h1>Gallery</h1>
-          <div class="gallery-container-outer">
-            <div class="gallery-container-inner" fxLayoutAlign="center center">
-              <ngx-gallery [options]="galleryOptions" [images]="galleryImages">
-              </ngx-gallery>
-
-            </div>
-        </div>
-      </div>
-
       <!-- Project showcase -->
       <div class="section">
         <h1>Projects</h1>
 
-        <div fxLayout="column" fxLayoutAlign='center'>
-          <div fxLayout="row" fxLayoutAlign='center'>
-
-            <div fxFlex="0 0 60%" fxLayout="column">
-              <h2><a href="https://github.com/gazebosim/gz-omni">Omniverse</a></h2>
+        <div fxLayout="row" fxLayoutAlign='center' fxLayoutGap="20px">
+          <mat-card>
+            <a routerLink="/projects/maritime">
+            <img mat-card-image src='assets/images/projects/sydney_regatta.png'>
+            </a>
+            <mat-card-header>
+              <mat-card-title>Maritime simulation</mat-card-title>
+            </mat-card-header>
+            <mat-card-content>
               <p>
               This project linked Gazebo with NVIDIA Omniverse through a <a
```

Comment:
Should we replace this block with a summary of the maritime project?

Reply:
Ahh, right. That was a place holder. Do you have a summery available?

Reply:
Something like this:

"Learn about the available Gazebo resources that let you create maritime simulations. Underwater and surface simulations are possible by adding sensors and plugins to the models in your world."